src/convertToObject.js

```diff
@@ -12,7 +12,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const styles = sourceString.split(';');
+
+  return styles.reduce((prev, el) => {
+    if (el.trim() !== '' && el.trim() !== ' ') {
```

Comment:
do you really need this check?` el.trim() !== '   '`

Reply:
Oh, I think no need, thank you)